### DISCUSSION TITLE
Fixed - Please ignore or delete message

### DISCUSSION THREAD

**Discussion by markkkky** [GerryDazoo/Slinger#48] [Q&A]
Firstly thank you so much for this, but I was struggling to get this to work on my local network (haven't tried remotely yet but will do this after), but now it IS working. I thought I had a SOLO, but it was a M1 so changing in config has now got this working. Will try remotely shortly. 

Superb work from Accomplished-Sign428  Thank you!

**Comment by GerryDazoo** [comment]
VLC is unable to open the MRL 'http://192.168.0.9:8081/slingbox'. Check the log for details.

 

'http://192.168.0.9 <http://192.168.0.9:8080/slingbox> :8080/slingbox'.

**Reply by markkkky** [reply]
Hi Gerry, many thanks for your answer. I have since fixed this. I am now working on the remote for my Sky TV box as some of the buttons are not working. Regards M

**Comment by GerryDazoo** [comment]
Great,

 

Here’s an update on how to “easily” find your key codes compared to my tiral and error method.

 

If you have a prohd or solo, install slingplayer 2.0.4. Open sbcore.ini located in C:\ProgramData\Sling Media\SlingPlayer\2.0 and add DebugRemote=1 under [main] Then, when you hover over each button on the virtual remote, it will show the mapped key code.

 

I think this will work for the 500 too.

 

Also, you can clear up all that probing for the slingbox by adding the correct port number in the config.ini

port=5301

**Reply by markkkky** [reply]
Hi Gerry, I've downloaded it, but it doesn't support my M1 box :-(

Here is the remote;
![Slingbox Sky HD TV Remote](https://user-images.githubusercontent.com/90906341/189478081-7f379088-c846-446f-8216-fafbe0ab3b70.PNG)

The buttons that work ok are;

0, 1, 2, 3, 4, 5, 6, 7, 8, 9 all OK
Up arrow OK
down arrow OK
left arrow OK
right arrow OK
select = OK
guide OK
record OK
play/pause OK
fast forward OK
rewind OK
backup = exit OK
ch + = OK
ch - = OK


I will also need to work out the color buttons, then once I get a better understanding of how to do this, will start on all the other buttons.

Is there a procedure I need to follow, to get this worked out

Many thanks for any pointers....

M
